### ISSUE TITLE
Existing Treasure Box

### ISSUE THREAD

**Issue by BILINGBALL** [zerodytrash/TikTok-Live-Connector#89]
Is it possible to detect the already existing envelop chest?

**Comment by Cojad** [comment]
I don't think you are likely getting an answer. This tool is not to mean to scan the treasure chest. Good luck on your search.

**Comment by ahmeddaggart** [comment]
كيف يمكنني اختراق صندوق الكنز في تيك توك

**Comment by CehennemGibiyim** [comment]
It is actually possible to do these, but I need someone who can help me :)
